save-frontend/src/main/kotlin/org/cqfn/save/frontend/components/views/OrganizationView.kt

```diff
@@ -448,10 +493,35 @@ class OrganizationView : AbstractView<OrganizationProps, OrganizationViewState>(
         }
     }
 
+    private fun setNewDescription(value: String) {
+        descriptionTmp = value
+    }
+
+    private fun onOrganizationSave() {
+        setState {
+            organization?.apply {
```

Comment:
Will it actually cause state change? I thought, state property needs to be assigned with a new value

Reply:
Yes, this update will change the state

Reply:
In pure React `setState` accepts an instance of state object; but I forgot that in Kotlin there is an extension function, that accepts a lambda.